api/envoy/extensions/quic/v3/crypto_stream.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package envoy.extensions.quic.v3;
+
+import "udpa/annotations/status.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.quic.v3";
+option java_outer_classname = "CryptoStreamProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: QUIC server crypto stream config]
+// [#extension: quic.quiche_crypto_server_stream]
```

Comment:
I think the extension name should start with `envoy.quic.server.crypto_stream` (the category defined above).

Reply:
done